app/app.go

```diff
@@ -433,56 +445,87 @@ func NewDesmosApp(
 	app.CapabilityKeeper.Seal()
 
 	// add keepers
-	app.AccountKeeper = authkeeper.NewAccountKeeper(
-		appCodec, keys[authtypes.StoreKey], app.GetSubspace(authtypes.ModuleName), authtypes.ProtoBaseAccount, maccPerms,
-	)
+	app.AccountKeeper = authkeeper.NewAccountKeeper(appCodec, keys[authtypes.StoreKey], authtypes.ProtoBaseAccount, maccPerms, sdk.Bech32MainPrefix, authtypes.NewModuleAddress(govtypes.ModuleName).String())
```

Comment:
In 0.47.0, cosmos sdk modules like `x/auth`, `x/bank` and so on, are not depended on `x/params` anymore. They will handle params change proposal direction inside their own module controlled byx `x/gov`.